server/api/db/sqldb/db.py

```diff
@@ -1922,29 +1922,42 @@ def update_function_external_invocation_url(
             )
             return
 
+        # remove trailing slashes from the URL
+        url = url.rstrip("/")
+
```

Comment:
I think this should be done on client side as well when we generate `invoke_url`